.github/workflows/run-host-tests.yml

```diff
@@ -56,7 +74,12 @@ jobs:
             apt-get update && apt-get install -y python3-pip rsync
             python -m pip install gcovr
             cd $GITHUB_WORKSPACE/${{inputs.component_path}}
-            gcov `find . -name "*gcda"`
+            component=$(basename ${{ inputs.component_path }})
+            if [[ ${{ inputs.idf_version }} == "release-v4.3" ]]; then
+                gcov-8 `find . -name "$component*gcda" -printf '%h\n' | head -n 1`/*
+            else
+                gcov `find . -name "$component*gcda"`
+            fi
```

Comment:
TBH, I don't very much like this condition. could be decide based on `gcov` version, rather than IDF version?

Reply:
Thank you for the suggestion. The problem is, for the `idf-version-4.3` gcov will not work, as it is linked to gcov-9, which failed, in order to build for idf-version-4.3 `gcov-8` is needed